### ISSUE TITLE
Published JARs for versions 2.2.0 and 2.3.0 have an invalid module name in MANIFEST.MF

### ISSUE THREAD

**Issue by anderswisch** [bbottema/email-rfc2822-validator#24]
Published JARs for versions 2.2.0 and 2.3.0 have an invalid module name in `MANIFEST.MF`.

Both contain this line:

```
Automatic-Module-Name: emailaddress-rfc2822
```

Because it contains a `-` character, it is not a valid identifier. Running `jar --describe-module --file emailaddress-rfc2822-2.2.0.jar` gives output:

```
Unable to derive module descriptor for: emailaddress-rfc2822-2.2.0.jar
Automatic-Module-Name: emailaddress-rfc2822: Invalid module name: 'emailaddress-rfc2822' is not a Java identifier
```

**Comment by anderswisch** [comment]
I think this comes from https://github.com/bbottema/standard-project-parent/blob/master/pom.xml#L60 (`artifactId` can contain dashes)

**Comment by anderswisch** [comment]
This problem also affects version 1.0.1 of https://github.com/bbottema/jetbrains-runtime-nullability-annotations

**Comment by bbottema** [comment]
Hmm, that's a problem. I'm not sure how to get the unique project name without dashes and I don't really have the time lately to do research. Any suggestions?

**Comment by anderswisch** [comment]
I put up a clumsy fix for this in https://github.com/bbottema/standard-project-parent/pull/10. Let me know what you think. It's been awhile since I tried to do this kind of thing in Maven.

**Comment by bbottema** [comment]
Fixed by https://github.com/bbottema/email-rfc2822-validator/commit/aea17f8b08638c5ee21e93f0f1335c6f52471d51

**Comment by bbottema** [comment]
Fix released in 2.3.1
